src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/StationaryCameraPoseProvider.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.MixedReality.SpatialAlignment;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using Stopwatch = System.Diagnostics.Stopwatch;
+
+namespace Microsoft.MixedReality.SpectatorView
+{
+    public class StationaryCameraPoseProvider : MonoBehaviour
+    {
+        private INetworkManager networkManager;
+        private Stopwatch timestampStopwatch;
+        private SpatialCoordinateSystemParticipant sharedCoordinateParticipant;
+        private INetworkConnection currentConnection;
+
+        private void Awake()
+        {
+            networkManager = GetComponent<INetworkManager>();
+            if (networkManager == null)
+            {
+                throw new MissingComponentException("Missing network manager component");
+            }
+
+            networkManager.Connected += NetworkManagerConnected;
+
+            if (networkManager.IsConnected)
+            {
+                var connections = networkManager.Connections;
+                if (connections.Count > 1)
+                {
+                    Debug.LogWarning("More than one connection was found, CameraPoseProvider only expects one network connection");
+                }
+
+                foreach (var connection in connections)
+                {
+                    NetworkManagerConnected(connection);
+                }
+            }
+        }
+
+        private void OnDestroy()
+        {
+            networkManager.Connected -= NetworkManagerConnected;
+        }
+
+        private void Update()
+        {
+            if (currentConnection == null || !currentConnection.IsConnected)
+            {
+                return;
+            }
+
+            if (sharedCoordinateParticipant == null)
+            {
+                SpatialCoordinateSystemManager.Instance.TryGetSpatialCoordinateSystemParticipant(currentConnection, out sharedCoordinateParticipant);
+            }
```

Comment:
Does this mean that a stationary camera always reports its location as the location of the qr code/aruco marker in the scene?